### PR TITLE
Add fallback support for section1 prototype

### PR DESCRIPTION
**Pull request by andrewadcock**
Adds support for the legacy /section1

### CODE REVIEW

**Comment by andrewadcock** [comment]
Updated with changes suggested by @tadhg-ohiggins

File: docs/section-schemas/backend-json-section-3.json [hunk 1]
```diff
@@ -409,18 +382,15 @@
                 ],
                 "context_data": {
                   "conditional_display": {
+                    "toMatch": { "no": "2020-03-d-01-01" },
```
Comment (tadhg-ohiggins) [inline docs/section-schemas/backend-json-section-3.json:385]: Can you remove this line? The JSON won't validate if it's there. @andrewadcock

**Review by tadhg-ohiggins — CHANGES_REQUESTED** [review]
Overall, I think that most of these changes look like whitespace tweaks, and I think we should probably revert those. I think that ideally this PR shouldn't need any changes to the JSON docs; let me know if there is a necessary change that I've missed.

**Comment by andrewadcock** [comment]
The whitespace changes came from my linter. I reverted back to the original though.

**Comment by tadhg-ohiggins** [comment]
I'm still seeing a `toMatch` in there; should be good to go once they're all removed.

**Comment by andrewadcock** [comment]
I believe I did it right this time. All the toMatch should be removed.

**Review by tadhg-ohiggins — APPROVED** [review]

**Pull request merged**